src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/importer/price/PricesSpreadsheet.kt

```diff
@@ -58,20 +59,23 @@ class PricesSpreadsheet(
     val toLocation = locations[toLocationName]
       ?: throw RuntimeException("To location '$toLocationName' for supplier '$supplier' not found")
 
-    // Note: we do not need to worry about the effective year as we always start on a clean slate on import.
-    pricesRepository.findBySupplierAndFromLocationAndToLocation(supplier, fromLocation, toLocation)?.let {
-      throw RuntimeException("Duplicate price: '${fromLocation.siteName}' to '${toLocation.siteName}' for $supplier")
-    }
+    failIfPriceAlreadyExists(supplier, fromLocation, toLocation)
 
     return Price(
       supplier = supplier,
       fromLocation = fromLocation,
       toLocation = toLocation,
       priceInPence = price,
-      effectiveYear = 2020 // TODO remove this once we're not importing from spreadsheet anymore
```

Comment:
The reason this was here is down to the fact that price importing manually was supposed to be short lived functionality to help the end users i.e. no longer applicable.  However uptake/use of the system has been slow.